km62/Yevlentiev_Mykhailo/laboratory1.sql

```diff
@@ -1,2 +1,134 @@
 -- LABORATORY WORK 1
 -- BY Yevlentiev_Mykhailo
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+створювати таблиці та видаляти з них дані
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE USER evlentiev IDENTIFIED BY love_sql
+DEFAULT TABLESPACE "USERS" TO evlentiev
+TEMPERARY TABLESPACE "TEMP" TO evlentiev
+
+ALTER "CONNECT" TO evlentiev
+ALTER CREATE ANY TABLE TO evlentiev
```

Comment:
Instead of "ALTER" there should be "GRANT"
Forgot ";"

Must be:
GRANT CREATE ANY TABLE TO evlentiev;